inst/include/interface/rcpp/rcpp_objects/rcpp_data.hpp

```diff
@@ -80,6 +80,26 @@ class AgeCompDataInterface : public DataInterfaceBase {
   /** @brief get the ID of the interface base object
    */
   virtual uint32_t get_id() { return this->id; }
+  
+  
+  virtual std::string to_json() {
```

Comment:
@msupernaw how should to_json() be documented here? Maybe
```
  /** @brief Get information for json output.
   */
```

Reply:
Moving Matthew's response here ... hmm, good question. I think some sort of blurb about converting the data to a json representation to be used in the output?